### PR TITLE
Timing, Laser and Hot cell quality plots adjustments [Master]

### PR DESCRIPTION
**Pull request by abhih1**
#### PR description:

This PR does the following in ECAL DQM plots:
 1. Changes the threshold in Timing quality summary to reduce the false positives in Ecal Endcaps
 2. Changes the amplitude threshold in laser quality for the forward region of Ecal Endcaps
 3. Disables the condition for setting entire FED to RED in Hot Cell quality as it's creating false alarms in the presence of single hot towers.
 
#### PR validation:
Validated by running the standard online Ecal DQM configuration and checking the output on a test DQM GUI. 

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Backported to 12_4_X here:  https://github.com/cms-sw/cmssw/pull/39162
Backported to 12_5_X here: https://github.com/cms-sw/cmssw/pull/39327

### CODE REVIEW

**Comment by emanueleusai** [comment]
please test

**Comment by abhih1** [comment]
The error is unrelated to the changes in the PR

**Comment by emanueleusai** [comment]
+1

- changes in ECALBarrel folder as expected
- error in unit test src/Configuration/PyReleaseValidation/test/test-das-selected-lumis unrelated to this PR ( https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-2185be/27038/unitTests/src/Configuration/PyReleaseValidation/test/test-das-selected-lumis/testing.log )

File: DQM/EcalMonitorClient/interface/TimingClient.h [hunk 1]
```diff
@@ -15,7 +15,8 @@ namespace ecaldqm {
   private:
     void setParams(edm::ParameterSet const&) override;
 
-    float toleranceMean_;
+    float EBtoleranceMean_;
```
Comment (perrotta) [inline DQM/EcalMonitorClient/interface/TimingClient.h:18]: Please modify the class member with lowercase initial, e.g. ebToleranceMean_ and eeToleranceMean_

File: DQM/EcalMonitorClient/python/TimingClient_cfi.py [hunk 2]
```diff
@@ -5,7 +5,8 @@
 
 minChannelEntries = 1
 minTowerEntries = 3
-toleranceMean = 2.
+EBtoleranceMean = 2.
```
Comment (perrotta) [inline DQM/EcalMonitorClient/python/TimingClient_cfi.py:8]: Please modify the configuration parameters names with lowercase initials

File: DQM/EcalMonitorClient/interface/TimingClient.h (hunk 1 above)
Comment (abhih1) [reply DQM/EcalMonitorClient/interface/TimingClient.h]: Sure.

**Comment by perrotta** [comment]
please test

**Comment by emanueleusai** [comment]
tests at P5 of the backport ok

**Comment by emanueleusai** [comment]
+1

**Comment by perrotta** [comment]
+1

**Pull request merged**